### PR TITLE
AB#97232 Adjusted user organisation to display ahp projects instead o…

### PR DESCRIPTION
**Pull request by MichalSzrederPWC**
### 🛠 Changes being made

Changed user organisation view to display ahp projects instead applications

### 📸 Screenshots (optional)

image
image

### 🏎 Quality check

- [x] Have you performed local tests?
- [ ] Are integration tests passing locally?
- [ ] Have you added some unit tests?
- [ ] Have you added some integration tests?
- [x] Have you checked WCAG (included screenshot)

### CODE REVIEW

File: src/HE.Investments.Account.Domain/UserOrganisation/Repositories/ProgrammeApplicationsRepository.cs [hunk 1]
```diff
@@ -76,31 +75,29 @@ private async Task<IList<UserApplication>> GetLoansApplications(UserAccount user
...
-                HeApplicationId.From(a.loanApplicationId),
+        return loanApplications.Select(a => new UserAppliance(
+                HeApplianceId.From(a.loanApplicationId),
                 a.ApplicationName,
                 ApplicationStatusMapper.MapToPortalStatus(a.loanApplicationExternalStatus)))
             .ToList();
     }
 
-    private async Task<IList<UserApplication>> GetAhpApplications(UserAccount userAccount, CancellationToken cancellationToken)
+    private async Task<IList<UserAppliance>> GetAhpProgrammes(UserAccount userAccount, CancellationToken cancellationToken)
```
Comment (jjankowski87) [inline src/HE.Investments.Account.Domain/UserOrganisation/Repositories/ProgrammeApplicationsRepository.cs:85]: It's not programmes but projects

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investments.Account.Contract/UserOrganisation/UserAppliance.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using HE.Investments.Common.Contract;
+
+namespace HE.Investments.Account.Contract.UserOrganisation;
+
+public class UserAppliance
```
Comment (Carq) [inline src/HE.Investments.Account.Contract/UserOrganisation/UserAppliance.cs:5]: What is `Appliance` ?

**Review by Carq — APPROVED** [review]

File: src/HE.Investments.Account.Contract/UserOrganisation/UserAppliance.cs (hunk 2 above)
Comment (MichalSzrederPWC) [reply src/HE.Investments.Account.Contract/UserOrganisation/UserAppliance.cs:5]: Application or Project, although I'm open to other suggestions

File: src/HE.Investments.Account.Domain/UserOrganisation/Repositories/ProgrammeApplicationsRepository.cs (hunk 1 above)
Comment (MichalSzrederPWC) [reply src/HE.Investments.Account.Domain/UserOrganisation/Repositories/ProgrammeApplicationsRepository.cs]: done

**Pull request merged**